osu.Game.Modes.Osu/Objects/Drawables/Pieces/SliderBall.cs

```diff
@@ -98,17 +98,18 @@ protected bool Tracking
             }
         }
 
+        private bool validTrackingTime => Time.Current >= slider.StartTime && Time.Current <= slider.EndTime;

```

Comment:
I think the name should rather be `CanCurrentlyTrack` or something else that indicates the returned value is in fact _not_ a time, but a boolean.